reference-implementations/aegis128l.zig

```diff
@@ -229,5 +229,25 @@ fn Aegis128L_(comptime tag_bits: u9) type {
                 @memcpy(out[i..], aegis.enc(&zero)[0 .. out.len % 32]);
             }
         }
+
+        pub fn mac(
+            ad: []const u8,
```

Comment:
Should `ad` be renamed to match`data` in all these functions?

Reply:
Of course. It should match the spec at much as possible.

Reply:
By the way, I don't know if `data` is the best parameter name here.

Technically, this is handled like `ad`. Inputs to MACs are typically referred to as "messages". But that would be confusing as we use `msg` for something else everywhere else.

Reply:
Yeah, I think `data` works for that reason. Can't think of a better alternative.